packages/create-sitecore-jss/src/templates/nextjs-sxa/src/pages/api/robots.ts

```diff
@@ -1,16 +1,25 @@
 import type { NextApiRequest, NextApiResponse } from 'next';
 import config from 'temp/config';
-import { GraphQLRobotsService } from '@sitecore-jss/sitecore-jss-nextjs';
+import { GraphQLRobotsService, SiteResolver } from '@sitecore-jss/sitecore-jss-nextjs';
 
-const robotsApi = async (_req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+const robotsApi = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
   // Ensure response is text/html
   res.setHeader('Content-Type', 'text/html;charset=utf-8');
 
+  const hostDetails = {
```

Comment:
Might be good to add a comment to these new lines to explain. e.g. "// Resolve site based on host"